src/transaction/transaction.rs

```diff
@@ -43,12 +45,13 @@ use tikv_client_proto::{kvrpcpb, pdpb::Timestamp};
 /// # });
 /// ```
 pub struct Transaction {
-    status: TransactionStatus,
+    status: Arc<RwLock<TransactionStatus>>,
     timestamp: Timestamp,
     buffer: Buffer,
     bg_worker: ThreadPool,
     rpc: Arc<PdRpcClient>,
     options: TransactionOptions,
+    spawn_heartbeat: bool,
```

Comment:
I think there exists a less ambiguous name. Like `started_heartbeat` or something else

Reply:
what about `is_heartbeat_started`?

Reply:
what about `heartbeat_started`?

Reply:
It's a convention to give the bool variable a `is` prefix.

Reply:
For rust convention?